docs/data/data-grid/list-view/list-view.md

```diff
@@ -16,14 +16,107 @@ Unlike the default grid view, the list view makes no assumptions on how data is
 
 In order to display data in a list view, a `unstable_listColumn` prop must be provided with a `renderCell` function.
 
-{{"demo": "ListView.js", "bg": "inline"}}
+```tsx
+function ListViewCell(params: GridRenderCellParams) {
+  return <>{params.row.id}</>;
+}
+
+const listColDef: GridListColDef = {
+  field: 'listColumn',
+  renderCell: ListViewCell,
+};
+```
+
+{{"demo": "ListView.js", "bg": true}}
+
+## Enable with a media query
+
+The `useMediaQuery` hook from `@mui/material` can be used to enable the list view feature at a specified breakpoint.
+
+```tsx
+import * as React from 'react';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+function App() {
+  const theme = useTheme();
+  const isListView = useMediaQuery(theme.breakpoints.down('md'));
+
+  return <DataGrid unstable_listView={isListView} />;
+}
+```
+
+## Editable rows
+
+The [editing](/x/react-data-grid/editing/) feature is not supported in list view, however, it is possible to build an editing experience from within your custom cell renderer.
```

Comment:
- best practice: longer anchor text
- "but" is usually always better than "however" (it's more concise and less formal)
- point out that the demo that follows shows how to do this

```suggestion
The [editing feature](/x/react-data-grid/editing/) is not supported in list view, but it is possible to build an editing experience from within your custom cell renderer, as shown below.
```